src/api/routes/study.routes.js

```
@@ -1,22 +1,34 @@
 // Description: API를 위한 라우터 설정 코드 파일입니다.
+// 라이브러리 정의
 import express from 'express';
 import { Prisma } from '@prisma/client';
-import coreMiddleware from '../../../src/common/cors.js';
+
+// 미들웨어 정의
+import corsMiddleware from '../../../src/common/cors.js';
```

ESLint

[error] Unexpected use of file extension "js" for "../../../src/common/cors.js"

(import/extensions)